src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,55 @@
+.calendar {
```

Comment:
add some space above to prevent it:
![image](https://github.com/mate-academy/layout_calendar/assets/86923089/c27b43f9-c806-482b-9907-4d31a8abfecf)